assembly/Cargo.toml

```diff
@@ -20,6 +20,6 @@ default = ["std"]
 std = ["vm-core/std"]
 
 [dependencies]
-crypto = { package = "winter-crypto", version = "0.4.2", default-features = false }
+crypto = { package = "winter-crypto", version = "0.5.1", default-features = false }
```

Comment:
Why do we need it here? Could we not get everything from `miden-crypto` (maybe via `miden-core`)?